<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>848</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des etudiants</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
	background-color: #fff;
	color: red;

}


/*-----QLabel-----*/
QLabel
{
	background-color: transparent;
	color: #454544;
	font-weight: bold;
	font-size: 13px;

}


/*-----QPushButton-----*/
QPushButton
{
	background-color: #5c55e9;
	color: #fff;
	font-size: 13px;
	font-weight: bold;
	border-top-right-radius: 15px;
	border-top-left-radius: 0px;
	border-bottom-right-radius: 0px;
	border-bottom-left-radius: 15px;
	padding: 10px;

}


QPushButton::disabled
{
	background-color: #5c5c5c;

}


QPushButton::hover
{
	background-color: #5564f2;

}


QPushButton::pressed
{
	background-color: #3d4ef2;

}


/*-----QCheckBox-----*/
QCheckBox
{
	background-color: transparent;
	color: #5c55e9;
	font-size: 10px;
	font-weight: bold;
	border: none;
	border-radius: 5px;

}


/*-----QCheckBox-----*/
QCheckBox::indicator
{
    background-color: #323232;
    border: 1px solid darkgray;
    width: 12px;
    height: 12px;

}


QCheckBox::indicator:checked
{
    image:url(&quot;./ressources/check.png&quot;);
	background-color: #5c55e9;
    border: 1px solid #5c55e9;

}


QCheckBox::indicator:unchecked:hover
{
    border: 1px solid #5c55e9;

}


QCheckBox::disabled
{
	color: #656565;

}


QCheckBox::indicator:disabled
{
	background-color: #656565;
	color: #656565;
    border: 1px solid #656565;

}


/*-----QLineEdit-----*/
QLineEdit
{
	background-color: #c2c7d5;
	color: #2a547f;
	border: none;
	padding: 5px;

}


/*-----QListView-----*/
QListView
{
	background-color: #5c55e9;
	color: #fff;
	font-size: 14px;
	font-weight: bold;
	show-decoration-selected: 0;
	border-radius: 4px;
	padding-left: -15px;
	padding-right: -15px;
	padding-top: 5px;

} 


QListView:disabled 
{
	background-color: #5c5c5c;

}


QListView::item
{
	background-color: #454e5e;
	border: none;
	padding: 10px;
	border-radius: 0px;
	padding-left : 10px;
	height: 32px;

}


QListView::item:selected
{
	color: #000;
	background-color: #fff;

}


QListView::item:!selected
{
	color:white;
	background-color: transparent;
	border: none;
	padding-left : 10px;

}


QListView::item:!selected:hover
{
	color: #fff;
	background-color: #5564f2;
	border: none;
	padding-left : 10px;

}


/*-----QTreeView-----*/
QTreeView 
{
	background-color: #fff;
	show-decoration-selected: 0;
	color: #454544;

}


QTreeView:disabled
{
	background-color: #242526;
	show-decoration-selected: 0;

}


QTreeView::item 
{
	border-top-color: transparent;
	border-bottom-color: transparent;

}


QTreeView::item:hover 
{
	background-color: #bcbdbb;
	color: #000;

}


QTreeView::item:selected 
{
	background-color: #5c55e9;
	color: #fff;

}


QTreeView::item:selected:active
{
	background-color: #5c55e9;
	color: #fff;

}


QTreeView::item:selected:disabled
{
	background-color: #525251;
	color: #656565;

}


QTreeView::branch:has-children:!has-siblings:closed,
QTreeView::branch:closed:has-children:has-siblings 
{
	image: url(://tree-closed.png);

}

QTreeView::branch:open:has-children:!has-siblings,
QTreeView::branch:open:has-children:has-siblings  
{
	image: url(://tree-open.png);

}


/*-----QTableView &amp; QTableWidget-----*/
QTableView
{
    background-color: #fff;
	border: 1px solid gray;
    color: #454544;
    gridline-color: gray;
    outline : 0;

}


QTableView::disabled
{
    background-color: #242526;
    border: 1px solid #32414B;
    color: #656565;
    gridline-color: #656565;
    outline : 0;

}


QTableView::item:hover 
{
    background-color: #bcbdbb;
    color: #000;

}


QTableView::item:selected 
{
	background-color: #5c55e9;
    color: #fff;

}


QTableView::item:selected:disabled
{
    background-color: #1a1b1c;
    border: 2px solid #525251;
    color: #656565;

}


QTableCornerButton::section
{
	background-color: #ced5e3;
	border: none;
    color: #fff;

}


QHeaderView::section
{
	color: #2a547f;
	border: 0px;
	background-color: #ced5e3;
	padding: 5px;

}


QHeaderView::section:disabled
{
    background-color: #525251;
    color: #656565;

}


QHeaderView::section:checked
{
    color: #fff;
    background-color: #5c55e9;

}


QHeaderView::section:checked:disabled
{
    color: #656565;
    background-color: #525251;

}


QHeaderView::section::vertical::first,
QHeaderView::section::vertical::only-one
{
    border-top: 1px solid #353635;

}


QHeaderView::section::vertical
{
    border-top: 1px solid #353635;

}


QHeaderView::section::horizontal::first,
QHeaderView::section::horizontal::only-one
{
    border-left: 1px solid #353635;

}


QHeaderView::section::horizontal
{
    border-left: 1px solid #353635;

}


/*-----QScrollBar-----*/
QScrollBar:horizontal 
{
    background-color: transparent;
    height: 8px;
    margin: 0px;
    padding: 0px;

}


QScrollBar::handle:horizontal 
{
    border: none;
	min-width: 100px;
    background-color: #7e92b7;

}


QScrollBar::add-line:horizontal, 
QScrollBar::sub-line:horizontal,
QScrollBar::add-page:horizontal, 
QScrollBar::sub-page:horizontal 
{
    width: 0px;
    background-color: #d8dce6;

}


QScrollBar:vertical 
{
    background-color: transparent;
    width: 8px;
    margin: 0;

}


QScrollBar::handle:vertical 
{
    border: none;
	min-height: 100px;
    background-color: #7e92b7;

}


QScrollBar::add-line:vertical, 
QScrollBar::sub-line:vertical,
QScrollBar::add-page:vertical, 
QScrollBar::sub-page:vertical 
{
    height: 0px;
    background-color: #d8dce6;

}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>831</width>
      <height>571</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>Income</string>
     </attribute>
     <widget class="QTabWidget" name="modifier">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>821</width>
        <height>541</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>3</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Ajouter</string>
       </attribute>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>60</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>ID:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>140</y>
          <width>55</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Date:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>220</y>
          <width>55</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Income:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_ID">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>60</y>
          <width>351</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_Nom">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>140</y>
          <width>351</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>exemple:20/12/2000</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_Prnom">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>220</y>
          <width>351</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_Ajouter">
        <property name="geometry">
         <rect>
          <x>590</x>
          <y>400</y>
          <width>91</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Ajouter</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Afficher</string>
       </attribute>
       <widget class="QTableView" name="tableView">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>741</width>
          <height>261</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_tri">
        <property name="geometry">
         <rect>
          <x>600</x>
          <y>290</y>
          <width>121</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>tri income(asc)</string>
        </property>
       </widget>
       <widget class="QTableView" name="tableView_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>380</y>
          <width>551</width>
          <height>111</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_rechercher">
        <property name="geometry">
         <rect>
          <x>640</x>
          <y>470</y>
          <width>91</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>rechercher</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_r">
        <property name="geometry">
         <rect>
          <x>590</x>
          <y>430</y>
          <width>221</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="tridescendant">
        <property name="geometry">
         <rect>
          <x>420</x>
          <y>290</y>
          <width>131</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>tri income(desc)</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pdf">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>290</y>
          <width>91</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>pdf</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Supprimer</string>
       </attribute>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>50</y>
          <width>91</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>ID:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_ID_Suppr">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>50</y>
          <width>451</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_Supprimer">
        <property name="geometry">
         <rect>
          <x>560</x>
          <y>360</y>
          <width>101</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Supprimer</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>modifier</string>
       </attribute>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>70</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>ID:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>140</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Date:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>200</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Income:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_ID_MOD">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>70</y>
          <width>541</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_NOM_MOD">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>140</y>
          <width>541</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_P_MOD">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>200</y>
          <width>541</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_modifier">
        <property name="geometry">
         <rect>
          <x>590</x>
          <y>280</y>
          <width>81</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>modifier</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_6">
     <attribute name="title">
      <string>Expenses</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>811</width>
        <height>511</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>3</number>
      </property>
      <widget class="QWidget" name="tab_7">
       <attribute name="title">
        <string>ajouter</string>
       </attribute>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>80</y>
          <width>51</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>ID:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>150</y>
          <width>61</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Date:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>220</y>
          <width>71</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Description:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>310</y>
          <width>61</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Expenses:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>80</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_2">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>150</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_3">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>230</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_4">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>310</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_ajout_expense">
        <property name="geometry">
         <rect>
          <x>530</x>
          <y>150</y>
          <width>121</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>ajout expense</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_8">
       <attribute name="title">
        <string>afficher</string>
       </attribute>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>80</y>
          <width>51</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QTableView" name="tableView_3">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>741</width>
          <height>261</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pdf_2">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>300</y>
          <width>101</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>pdf</string>
        </property>
       </widget>
       <widget class="QPushButton" name="tridescendant_2">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>300</y>
          <width>141</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>tri description</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_9">
       <attribute name="title">
        <string>supprimer</string>
       </attribute>
       <widget class="QLabel" name="label_13">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>80</y>
          <width>51</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>ID:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_5">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>80</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_suppression_expense">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>70</y>
          <width>121</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>supprimer</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_10">
       <attribute name="title">
        <string>modifier</string>
       </attribute>
       <widget class="QLabel" name="label_14">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>70</y>
          <width>51</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>ID:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_6">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>70</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_7">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>140</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_15">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>140</y>
          <width>61</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Date:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_16">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>210</y>
          <width>71</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Description:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_8">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>220</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_17">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>290</y>
          <width>61</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Expenses:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_9">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>290</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_modifier_expense">
        <property name="geometry">
         <rect>
          <x>440</x>
          <y>280</y>
          <width>131</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>modifier expense</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
